vispy/visuals/colorbar.py

```diff
@@ -204,6 +216,35 @@ def _prepare_draw(self, view):
         self._draw_mode = "triangles"
         return True
 
+    @property
+    def banded(self):
+        return self._banded
+
+    @banded.setter
+    def banded(self, banded):
+        self._banded = banded
+        self.shared_program.frag['color_transform'] = self.define_fun()
+        self._update()
+
+    @property
+    def nband(self):
+        return self._nband
+
+    @nband.setter
+    def nband(self, nband):
+        self._nband = nband
+        self.shared_program.frag['color_transform'] = self.define_fun()
+        self._update()
+
+    def define_fun(self):
```

Comment:
Can you rename this to something private and make it more clear what it is doing? Something like `_get_color_transform()`? Open to other names. Mainly just thinking that this doesn't need to be public and `define_fun` is not very clear.

Reply:
Ok i change this name